InstantSearch.js/refresh-cache-periodically/src/app.js

```diff
@@ -28,7 +28,7 @@ search.addWidget(
     container: '#hits',
     templates: {
       item: `
-        <div class="ais-Hits-item">
+        <div>
           <header class="hit-name">
             {{{_highlightResult.name.value}}}
```

Comment:
We should migrate to the highlight function.

Reply:
Done in dd77325fa5b30e7c54bb3786440cf9a730d66992.